### ISSUE TITLE
small typo fix: OU vs Pagel

### ISSUE THREAD

**Issue by cecileane** [lukejharmon/ilhabela#1]
Hi Luke,

I've seen people use `corPagel` instead of `corMartins` after following your instructions [here](https://lukejharmon.github.io/ilhabela/instruction/2015/07/03/PGLS/) (at the bottom). It looks like the confusion I've seen is from a simple typo in the text above your code, line 95 of [`PGLS.Rmd`](https://github.com/lukejharmon/ilhabela/blob/e18e94ef01a7c36f7fb0c7cae74067ddf826538e/workingFiles/PGLS/PGLS.Rmd#L95). Just replace "follows an OU model rather than Brownian motion" by "follows Pagel's lambda model rather than Brownian motion" (say).

And then add a comment within the code block below, [line 108](https://github.com/lukejharmon/ilhabela/blob/e18e94ef01a7c36f7fb0c7cae74067ddf826538e/workingFiles/PGLS/PGLS.Rmd#L108) saying that now we can use an error structure assuming the OU model.

Small things. For folks who use your nice tutorial.

**Comment by r03ert0** [comment]
Thank you @cecileane for the warning! Is it the case then that Martins & Hansen model is equivalent to the Ornstein-Uhlenbeck model?

**Comment by cecileane** [comment]
Yes indeed. The function [corMartins](https://www.rdocumentation.org/packages/ape/versions/5.4-1/topics/corMartins) implements the correlation structure for the OU process, when the ancestral state at the root is modeled as a random variable with the stationary distribution of the OU process. It's the same as the "OUrandomRoot" correlation model of the [phylolm](https://www.rdocumentation.org/packages/phylolm/versions/2.6.2/topics/phylolm) function.
